Applications/SystemMonitor/main.cpp

```diff
@@ -220,7 +221,7 @@ int main(int argc, char** argv)
             if (path.has_value())
                 Desktop::Launcher::open(URL::create_with_file_protocol(LexicalPath(path.value()).dirname()));
             else
-                GUI::MessageBox::show_error("Couldn't open path!");
+                GUI::MessageBox::show_error(String::format("Failed to open path: %s", strerror(errno)));
```

Comment:
Let's also specify which path was it that we tried and failed to open.

But the way, this clearly doesn't belong in the readlink commit...